keyboards/hs60/v2/rules.mk

```
@@ -50,7 +50,7 @@ DFU_ARGS = -d 0483:df11 -a 0 -s 0x08000000:leave
 #   comment out to disable the options.
 #
 BACKLIGHT_ENABLE = no              # Enable keyboard backlight functionality
-BOOTMAGIC_ENABLE = no              # Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = yes             # Virtual DIP switch configuration
```

Could you revert this? 

```suggestion
BOOTMAGIC_ENABLE = no             # Virtual DIP switch configuration
```

If needed, add this to your keymap's `rules.mk`.

Unless @yiancar is okay with this, that is.

---

moved it to the keycap rules.mk